tests/output-eve-anomaly/test.rules

```diff
@@ -0,0 +1 @@
+alert tcp any 20 -> any any (msg:"Simple rule"; filestore;  sid:31; rev:1;)
```

Comment:
no need to add a rules file if you're not checking the results or it matching anyway

Reply:
Will remove.